package.json

```
@@ -24,7 +24,7 @@
   "dependencies": {
     "@0xpolygonhermez/zkevm-commonjs": "github:0xpolygonhermez/zkevm-commonjs#v1.0.0",
     "@axelar-network/axelar-cgp-solidity": "6.3.1",
-    "@axelar-network/axelar-cgp-sui": "https://github.com/axelarnetwork/axelar-cgp-sui.git",
+    "@axelar-network/axelar-cgp-sui": "^0.2.0",
```

```suggestion
    "@axelar-network/axelar-cgp-sui": "0.2.0",
```
we upgrade our repo dependencies deliberately, to avoid including an unintentional change in the contracts